src/sections/SideBar/components/Footer/SideBarFooter.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { makeStyles, Link, Typography } from "@material-ui/core";
+import { styles } from "../../../../styles/styles";
```

Comment:
same here with the imports, see if we can decrease the amount of paths to /styles by adding some rule somewhere that knows importing from /styles means that route